tasks/sep/012-fp-fjord-upgrade-fix/VALIDATION.md

```diff
@@ -0,0 +1,26 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of the upgrade
+transaction.
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## State Changes
+
+The only pertinent state changes (ignoring safe nonce updates) should be made to the `DisputeGameFactoryProxy` game type implementations.
+
+### `0x05f9613adb30026ffd634f38e5c4dfd30a197fa1` (`DisputeGameFactoryProxy`)
+
+- **Key**: `0x4d5a9bd2e41301728d41c8e705190becb4e74abe869f75bdb405b63716a35f9e`
+  **Before**: `0x000000000000000000000000848e6ff026a56e75a1137f89f6286d14789997bc`
+  **After**: `0x000000000000000000000000864fbe13bb239521a8c837a0aa7c7122ee3eb0b2`
+  **Meaning**: Updates the PERMISSIONED_CANNON game type implementation. Verify that the new implementation is set using `cast call 0x05f9613adb30026ffd634f38e5c4dfd30a197fa1 gameImpls(uint32)(address) 1`.
```

Comment:
```suggestion
  **Meaning**: Updates the PERMISSIONED_CANNON game type implementation. Verify that the new implementation is set using `cast call 0x05f9613adb30026ffd634f38e5c4dfd30a197fa1 "gameImpls(uint32)(address)" 1`.
```

Also, ideally we should explain how to calculate the slot number—the `cast call` doesn't actually demonstrate that this key is the one we want to change